<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Try Catch</title>
</head>
<body>
    <script type="text/javascript">
    //try catch:在try里面的发生错误不会执行错误后的try里面的代码
    try{
        console.log('a');
        console.log(b);
        console.log('v');
    }catch(e){//error error.message error.name    e是形参可随便取值
        console.log(e.message + " " + e.name);
    }
    console.log("d");
    //error name 共6种
    //1.EvalError: eval()的使用预定义不一致
    //2.RangeError：

    </script>
</body>
</html>